packages/clay-dropdown/src/ClayDropdownBase.js

```diff
@@ -162,6 +163,15 @@ ClayDropdownBase.STATE = {
     'secondary',
     'unstyled',
   ]).value('unstyled'),
+
+  /**
+   * Type of the dropdown menu.
+   * @instance
+   * @memberof ClayDropdownBase
+   * @type {!string}
```

Comment:
Would it be optional or required? In `soy` is it optional.

Reply:
Is not required, wrong ! in @type